app/views/collections/permissions.html.erb

```diff
@@ -3,7 +3,7 @@
 <% if @collection.owned_by?(current_user) %>
   <h2 class = "center-text">Permissions for <em><%= @collection.title %></em></h2>
   <h3 class = "center-text">Visibility:
-    <% if @collection.private %>
+    <% if @collection.is_private %>
     Private (only visible to you)
       <%= form_for(@collection, url: "/collections/#{@collection.id}", method: 'put') do |f|%>
       <%= f.hidden_field :private, value: false %>
```

Comment:
I think we need to change the references in the permissions form from :private to :is_private, and also in the collections controller's valid_params method.